### PR TITLE
[PDI-15259] Wrong architecture name for FreeBSD in spoon.sh

### PR DESCRIPTION
**Pull request by smmribeiro**
Removed the case options that were superfluous as they all basically had the behaviour that the default case has.

@pentaho/tatooine @pentaho-lmartins @ppatricio

### CODE REVIEW

File: assemblies/core/static/src/main/resources-filtered/spoon.sh [hunk 1]
```diff
@@ -191,23 +191,11 @@ case `uname -s` in
...
-				LIBPATH=$CURRENTDIR/../libswt/linux/x86_64/
-				echo "I'm sorry, this FreeBSD platform [$ARCH] is not yet supported!"
-				exit
-				;;
-
 			i[3-6]86)
 				LIBPATH=$CURRENTDIR/../libswt/linux/x86/
 				;;
 
-			ppc)
```
Comment (pentaho-lmartins) [inline assemblies/core/static/src/main/resources-filtered/spoon.sh]: Hmmm.... reminds me of my old PowerPC some good 20 years ago...
Why are we removing this? The JIRA case says nothing about removing PPC.

I don't see SWT for PPC being included in the assemblies in https://github.com/pentaho/pentaho-kettle/blob/master/assemblies/client/src/assembly/assembly.xml. Maybe we should remove PPC from all operating systems.

File: assemblies/core/static/src/main/resources-filtered/spoon.sh [hunk 2]
```diff
@@ -191,23 +191,11 @@ case `uname -s` in
...
-				LIBPATH=$CURRENTDIR/../libswt/linux/ppc/
```
Comment (pentaho-lmartins) [inline assemblies/core/static/src/main/resources-filtered/spoon.sh]: In the PDI assembly we have under `libswt`:
- ./win32
- ./win64
- ./osx64
- ./linux/x86
- ./linux/x86_64

There are a few others that should be removed like `libswt/linux/ppc64/` and `libswt/solaris-x86/` above.

**Comment by buildguy** [comment]
### :gear: Build finished in 27m 3s with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -Dmaven-surefire-plugin.argLine=-XX:MaxRAMPercentage=90.0 -Dmaven-failsafe-plugin.argLine=-XX:MaxRAMPercentage=90.0
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4644|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/5/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 26m 39s with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -Dmaven-surefire-plugin.argLine=-XX:MaxRAMPercentage=90.0 -Dmaven-failsafe-plugin.argLine=-XX:MaxRAMPercentage=90.0
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4644|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/6/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 21m 50s with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -Dmaven-surefire-plugin.argLine=-XX:MaxRAMPercentage=90.0 -Dmaven-failsafe-plugin.argLine=-XX:MaxRAMPercentage=90.0
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4645|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/7/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 28m 54s with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -Dmaven-surefire-plugin.argLine=-XX:MaxRAMPercentage=90.0 -Dmaven-failsafe-plugin.argLine=-XX:MaxRAMPercentage=90.0
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 1 test(s) failed:
- **org.pentaho.di.trans.step.mqtt.MQTTStreamSourceTest.mqttReadsBinary**
```
    ${result.errorDetails}
```
  ---

|Total|Failed|Skipped|
|---|---|---|
|4643|1|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/8/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 33m 6s with command:
```
mvn clean verify -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -Dmaven-surefire-plugin.argLine=-XX:MaxRAMPercentage=90.0 -Dmaven-failsafe-plugin.argLine=-XX:MaxRAMPercentage=90.0
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 11 test(s) failed:
- **org.pentaho.di.trans.StepWithMappingMetaTest.loadMappingMeta**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.trans.StepWithMappingMetaTest.loadMappingMetaTest**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.trans.StepWithMappingMetaTest.testFileNameAsVariable**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.trans.TransMetaTest.testSetInternalEntryCurrentDirectoryWithRepository**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.trans.TransMetaTest.testLoadXml**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.trans.TransMetaTest.testTransWithOneStepIsConsideredUsed**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.www.ExecuteJobServletTest.testExecuteJobServletTest**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.www.ExecuteJobServletTest.testExecuteJobServletTestCantFindDirectory**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.www.ExecuteJobServletTest.testExecuteJobServletTestJobNotFoundInDirectory**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.www.ExecuteJobServletTest.testExecuteJobServletTestCantFindRepository**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
- **org.pentaho.di.www.ExecuteJobServletTest.testExecuteJobServletTestWithUnauthorizedUser**
```
    javassist.CannotCompileException: by java.util.zip.ZipException: invalid distance too far back
```
  ---

|Total|Failed|Skipped|
|---|---|---|
|4652|11|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/10/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 30m 1s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4652|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/11/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 29m 30s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4652|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/12/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 30m 59s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4652|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/13/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 29m 51s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4651|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/14/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 29m 36s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4653|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/15/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 29m 28s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4654|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/16/testReport)|
|----|

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 29m 1s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4657|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/18/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 28m 47s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4670|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/19/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 31m 21s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4670|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/20/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 31m 27s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4670|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/21/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 3m 26s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|1115|0|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/22/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 29m 53s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|4672|0|7|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6788/23/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by pentaho-lmartins** [comment]
Closing, see JIRA case for details.

**Pull request closed without merging**